ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -56,13 +48,13 @@ class SecondaryNavBar extends React.Component {
     this.setState({ searchQueryStr: ev.target.value });
   }
 
-  handleUrlChanges() {
+  handleUrlChanges = () => {
     this.setState({
       searchQueryStr: getSearchStrFromUrl(),
     });
-  }
+  };
 
-  search(ev) {
+  search = ev => {
```

Comment:
This wasn't bound before it seems. Does it need to be? :)

Reply:
Yeah, I've been looking around for "gotchas" regarding this feature.  The only things I saw mentioned don't sound like real issues for us.  (Like these functions are only part of the instance, not the ``prototype``.)  Could be for non-react classes in some very specific cases.  

So, along those lines, it seems like we should perhaps migrate to using this format.  In these PRs, I've not migrated more than I needed to.  Though to keep function order and make reviews easier, and since the linter requires that these functions go before non-class-level functions, I did up to the last one I needed to.

However, I'm now thinking we should perhaps just convert ALL our functions to this format unless there's a specific need for a function NOT to be this way.  What do you both think?  :)